<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonCopyName">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>apellido</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditLastName"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonCopyLastName">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Correo</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditMail"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonCopyMail">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditPass"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButtonCopyPass">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Fecha</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditDate"/>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButtonCopyDate">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Amigo</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEditFriend"/>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButtonCopyFriend">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Ocupacion</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEditOccupation"/>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButtonCopyOccupation">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Ciudad</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineEditCity"/>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButtonCopyCity">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Calle</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEditStreet"/>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="pushButtonCopyStreet">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Ciudad Pago</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="lineEditCityPay"/>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pushButtonCopyCityPay">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>CP</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="lineEditCp"/>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="pushButtonCopyCp">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>phone Prefijo</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="lineEditPhonePre"/>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="pushButtonCopyPhonePre">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>phone</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="lineEditPhone"/>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="pushButtonCopyPhone">
     <property name="text">
      <string>COPIAR</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
